### ISSUE TITLE
Error messages when completing show command on zsh

### ISSUE THREAD

**Issue by avorima** [gopasspw/gopass#2005]
### Summary

Tab completion for the show command on zsh outputs the following error messages:

```
_describe:compdescribe:124: invalid argument: subcommands
_describe:compdescribe:129: no parsed state
```

This happens when:
- no character matches an existing secret or store
- more than 1 character matches an existing secret or store

The completion still works though.

### Steps To Reproduce

```
$ gopass show secret<Tab>
```

### Expected behavior

No error messages.

### Environment

- OS version: Linux 5.14.9-arch2-1
- Zsh version: zsh 5.8 (x86_64-pc-linux-gnu)
- gopass Version: gopass 1.12.8 (2021-08-28 14:31:40) go1.17 linux amd64
- Installation method: pacman (https://archlinux.org/packages/community/x86_64/gopass/)

**Comment by newzubakhin** [comment]
The same for lots of subcommands, e.g. `gopass clone`
```
❯ gopass clone
_arguments:comparguments:325: invalid option definition: --crypto[Select crypto backend [age gpgcli plain]]
_arguments:comparguments:325: invalid option definition: --crypto[Select crypto backend [age gpgcli plain]]
_arguments:comparguments:325: invalid option definition: --crypto[Select crypto backend [age gpgcli plain]]
```
It looks like the syntax of _arguments is incorrect

**Comment by AnomalRoil** [comment]
Should be solved by #2044 I guess.

**Comment by avorima** [comment]
Version `gopass 1.14.2 go1.18.2 linux amd64` does not have any error messages anymore.
I would consider this as fixed.
